src/components/inputs/reactHookForm/utils/HelperPreviousValue.tsx

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) 2025, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { useIntl } from 'react-intl';
+import { FormHelperText, Stack, Tooltip, Typography } from '@mui/material';
+import { InfoOutlined, WarningAmberRounded } from '@mui/icons-material';
+
+export interface HelperPreviousValueProps {
+    previousValue?: number | string;
```

Comment:
we have also import { Input } from '../../../../utils/types/types'; can be reused for this props
```suggestion
    previousValue?: Input;
```

Reply:
ok